### ISSUE TITLE
unable to install the provider for 1.8.0

### ISSUE THREAD

**Issue by stephencoe** [instaclustr/terraform-provider-instaclustr#68]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description
I see you have recently started supporting terraform 0.13, the current version 1.8 doesn't resolve. It looks like the published version in the registry doesn't match the SHA in github. 

The registry SHA is `terraform-provider-instaclustr_1.8.0_SHA256SUMS`, github version is `terraform-provider-instaclustr_1.8_SHA256SUMS`

### Terraform Version

```
terraform --version
Terraform v0.13.5
```

### Terraform Configuration Files

```hcl

terraform {
  required_providers {
    instaclustr = {
      source = "registry.terraform.io/instaclustr/instaclustr"
      version = ">=1.8.0"
    }
  }
}
provider "instaclustr" {
  username = "foo"
  api_key  = "bar"
}

```

### Debug Output
```
2021/03/03 17:10:02 [TRACE] HTTP client GET request to https://registry.terraform.io/.well-known/terraform.json
- Finding instaclustr/instaclustr versions matching ">= 1.8.*"...
2021/03/03 17:10:03 [DEBUG] GET https://registry.terraform.io/v1/providers/instaclustr/instaclustr/versions
2021/03/03 17:10:03 [TRACE] HTTP client GET request to https://registry.terraform.io/v1/providers/instaclustr/instaclustr/versions
2021/03/03 17:10:03 [DEBUG] GET https://registry.terraform.io/v1/providers/instaclustr/instaclustr/1.8.0/download/darwin/amd64
2021/03/03 17:10:03 [TRACE] HTTP client GET request to https://registry.terraform.io/v1/providers/instaclustr/instaclustr/1.8.0/download/darwin/amd64
2021/03/03 17:10:03 [DEBUG] GET https://github.com/instaclustr/terraform-provider-instaclustr/releases/download/v1.8/terraform-provider-instaclustr_1.8.0_SHA256SUMS
2021/03/03 17:10:03 [TRACE] HTTP client GET request to https://github.com/instaclustr/terraform-provider-instaclustr/releases/download/v1.8/terraform-provider-instaclustr_1.8.0_SHA256SUMS
```

### Expected Behavior
installed the provider.

### Actual Behavior

```
Error: Failed to install provider

Error while installing instaclustr/instaclustr v1.8.0: could not query
provider registry for registry.terraform.io/instaclustr/instaclustr: failed to
retrieve authentication checksums for provider: 404 Not Found

[terragrunt] 2021/03/03 17:10:04 Hit multiple errors:
exit status 1
2021/03/03 17:10:04 FATAL exit status 1
```

### Steps to Reproduce

1. `terraform init`

**Comment by hail-insta** [comment]
Thanks @stephencoe for raising this with us. We have identified the issue and we aim to bring out the fix today or early next week.
We currently have some work in the pipe which will improve our CI flow which will prevent similar issues from happening in the future.

**Comment by hail-insta** [comment]
We have brought out a fix and our Terraform Provider is available in release v1.8.1. Confirmed the installation of this version with Terraform v0.13.
```
❯ terraform init

Initializing the backend...

Initializing provider plugins...
- Finding instaclustr/instaclustr versions matching "1.8.1"...
- Installing instaclustr/instaclustr v1.8.1...
- Installed instaclustr/instaclustr v1.8.1 (self-signed, key ID 7C6D41C1B8E86C42)

Partner and community providers are signed by their developers.
If you'd like to know more about provider signing, you can read about it here:
https://www.terraform.io/docs/plugins/signing.html

Terraform has been successfully initialized!

You may now begin working with Terraform. Try running "terraform plan" to see
any changes that are required for your infrastructure. All Terraform commands
should now work.

If you ever set or change modules or backend configuration for Terraform,
rerun this command to reinitialize your working directory. If you forget, other
commands will detect it and remind you to do so if necessary.

❯ terraform --version

Your version of Terraform is out of date! The latest version
is 0.14.7. You can update by downloading from https://www.terraform.io/downloads.html
Terraform v0.13.0
+ provider registry.terraform.io/instaclustr/instaclustr v1.8.1
